### ISSUE TITLE
[Bug]: Error Code X-2 while sample comments are being displayed

### ISSUE THREAD

**Issue by gregovin** [ThioJoe/YT-Spammer-Purge#558]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I received error code X-2 while the samples where being printed after logging while scanning videos.

Looking into the logs I see a mystery unpaired } at the end

### Release version

2.15.0-Beta2

### Steps to reproduce

1.  Open the spammer purge app without a config. Originally I discovered this with a config, but was able to replicate it without
2.  Continue as user
3.  Scan Techlinked's most recent video, "Lol Eat it Facebook"(https://youtu.be/gj4x9YaUATU). I was also able to get this bug to occur by scanning recent videos for the Tim Traveler and Techlinked, but it seems inconsistent(as it did not occur while scanning fireship's channel).
5. Enter not your channel mode
6. Choose Autosmart mode and enter to start the scan. Save the comments to log

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1591, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1568, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1202, in primaryInstance
  File "Scripts\logging.py", line 134, in print_comments
UnboundLocalError: local variable 'spamThreadSamplesContent' referenced before assignment
------------------------------------------------
Error Message: local variable 'spamThreadSamplesContent' referenced before assignment

Error - Code: X-2 occurred. This is almost definitely my fault and requires patching (big bruh moment)
Please post a bug report on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
    (In the mean time, try using a previous release of the program.)
```

**Comment by KendallDoesCoding** [comment]
@gregovin Is [2.14.3](https://github.com/ThioJoe/YT-Spammer-Purge/releases/tag/v2.14.3) working for you?

**Comment by lloyddunamis** [comment]
Encountered this one on the same 2.15.0-Beta2 WinExe.
2.14.3 and [2.15.0-Beta1](https://github.com/ThioJoe/YT-Spammer-Purge/releases/tag/v2.14.3) seem to work for me.

**Comment by leemark00** [comment]
i suspect there are problems with spacial characters

**Comment by KendallDoesCoding** [comment]
> Encountered this one on the same 2.15.0-Beta2 WinExe. 2.14.3 and [2.15.0-Beta1](https://github.com/ThioJoe/YT-Spammer-Purge/releases/tag/v2.14.3) seem to work for me.

Ok, then its a issue with beta, @ThioJoe should be able to fix this.

**Comment by ThioJoe** [comment]
Yea this should be fixed with https://github.com/ThioJoe/YT-Spammer-Purge/commit/d08a2ddfc5dc290f4529f812333f3f33ac678ab6 for the next release

**Comment by ThioJoe** [comment]
Ok new release is out with the fix, 2.15.0-Beta3
